tests/test_rag.py

```
@@ -67,29 +69,59 @@ def setUpClass(cls):
 
         sources = [Source(file_path)]
 
-        self.kg.process_sources(sources)
+        self.kg_gemini.process_sources(sources)
+        input = "How many actors acted in a movie?"
+
+        chat = self.kg_gemini.chat_session()
+        answer = chat.send_message(input)
+        answer_relevancy_metric = AnswerRelevancyMetric(threshold=0.5)
+        answer_cp_metric = ContextualPrecisionMetric(threshold=0.5)
+        answer_crecall_metric = ContextualRecallMetric(threshold=0.5)
```

Ruff (0.8.2)

Local variable `answer_crecall_metric` is assigned to but never used

Remove assignment to unused variable `answer_crecall_metric`

(F841)